docs/base-account/improve-ux/spend-permissions.mdx

```diff
@@ -98,6 +96,15 @@ await Promise.all(
 );
 ```
 
+<Note>
+**About the `spendCalls` array**
+
+This array has 2 calls when submitting the permission onchain for *the first time*.
+When the permission is already registered onchain, this array has only 1 call (the `spend` call).
```

Comment:
i think we need to still emphasize that when there are 2 calls and devs are using eth_sendTransaction, they need to make sure the calls are sending in order